processing/format_qaqc_driver.py

```diff
@@ -0,0 +1,288 @@
+                        stop_run = True
+                if stop_run:
+                    break
+                # run tasks
+                while o_tasks:
+                    # upload_ids is a list of upload_id
+                    # that has the same token
+                    for upload_ids in grouped_tasks:
+                        is_qaqc_successful = True
+                        with mp.Pool(processes=num_processes) as pool:
+                            for upload_id in upload_ids:
+                                task = o_tasks.get(upload_id)
+                                token = task.uuid
+                                is_zip = '.zip' in task.filename
+                                log.write(
+                                    (f'Start run: log id {task.upload_id}, '
+                                     f'prior id: {task.prior_process_id}, '
+                                     f'zip id: {task.zip_process_id}, '
+                                     f'run type: {task.run_type}\n'))
+                                processes.append(
+                                    {'process':
+                                        pool.apply_async(upload_checks,
+                                                         (task.filename,
+                                                          task.upload_id,
+                                                          task.run_type,
+                                                          task.site_id,
+                                                          task.
+                                                          prior_process_id,
+                                                          task.zip_process_id,
+                                                          self.is_test)),
+                                     'runtime': 0,
+                                     'retry': 0,
+                                     'task': task})
+                            while processes:
+                                time.sleep(self.time_sleep)
+                                s_processes = []
+                                for p in processes:
+                                    try:
+                                        result = (p
+                                                .get('process')
+                                                .get(timeout=5))
+                                        (process_id,
+                                        is_upload_successful,
+                                        uuid) = result
+                                        s_tasks = {}
+                                        if uuid and is_upload_successful:
+                                            s_tasks, _ = \
+                                                self.get_new_upload_data(log,
+                                                                        True,
+                                                                        uuid)
```

Comment:
Something is not jiving with the db query in get_new_upload_data method. By setting is_qaqc_processor = True, the query filters out uploads by the qaqc_processor.